options.go

```diff
@@ -245,9 +255,15 @@ func NATManager(nm config.NATManagerC) Option {
 // NoListenAddrs will configure libp2p to not listen by default.
 //
 // This will both clear any configured listen addrs and prevent libp2p from
-// applying the default listen address option.
+// applying the default listen address option. It also disables relay, unless the
+// user explicitly specifies with an option, as the transport creates an implicit
+// listen address that would make the node diable through any relay it was connected to.
```

Comment:
nit: s/diable/dialable

Reply:
sure, will fix; typos are bugs of sorts :)